scripts/ci-unit-tests.sh

```diff
@@ -4,4 +4,10 @@ command_file="unit-tests-command.sh"
 chmod +x $command_file && \
 ./$command_file
 
+# Exit code of gradle test command output
+exitCode=$?
+
 rm $command_file
+
+# Exit with test command output to set the GH action's check run status
+exit $exitCode
```

Comment:
We don't really need to store the previous command exit code. 
As far as we are concatenated command on a single line and using `&&`, whenever a command doesn't return `0` (It was run properly without errors), the following commands are not run
```suggestion
./$command_file && \
rm $command_file
```

Reply:
Ah good suggestions. Then the last command to delete the temporary `unit-tests-command` won't be deleted. Will it be a problem?

Reply:
It shouldn't. It will be there on the docker machine that is created, but because on every CI-Job the docker machine is a new one, it is not going to affect